pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
     "pandas<=2.2.0,>=0.25.0",
     "numpy<=1.26.4,>=1.22.0",
     "ordered-set<=4.1.0,>=4.0.2",
-    "fugue<=0.8.7,>=0.8.7",
+    "fugue[sql,duckdb]==0.9.0.dev2",
```

Comment:
Are you planning to release 0.9.0 soon? Would this work with the older version `0.8.7`

Reply:
Yes, should release Fugue 0.9.0 by the end of next week.